### PR TITLE
Quick-migration-of-Epicea-to-Spec2

### PR DESCRIPTION
**Pull request by jecisc**
Do a quick migration of Epicea to Spec 2. 

The code is not as good as it could be but it is not worst than before and will allow us to revert Spec "1.5" to Spec 1.

### CODE REVIEW

**Comment by tinchodias** [comment]
Ehhh! "not worse than before", hah. It will be good to see it in Spec2. Good.

**Comment by jecisc** [comment]
@tinchodias I mean that moving Epicea from Spec "1.5" to 2 like this does not make it use the new Spec 2 ways. Not that the Spec 1 version was bad ;)

File: src/EpiceaBrowsers/EpDashboardPresenter.class.st [hunk 1]
```diff
@@ -17,6 +17,20 @@ Class {
 	#category : #'EpiceaBrowsers-UI-Dashboard'
 }
 
+{ #category : #specs }
+EpDashboardPresenter class >> defaultSpec [
+	^ SpBoxLayout newVertical
+		add: #currentSessionInfoPresenter;
+		add:
```
Comment (estebanlm) [inline src/EpiceaBrowsers/EpDashboardPresenter.class.st:24]: Hi, 

Sorry for arriving late (I'm having a huge queue of issues to review and slowly walking them). 
The usage of `#add:withConstraints:` to force a height is bad. 
Instead,  we use `#add:expand:fill:padding:`. In this example the code will be: 

```Smalltalk
EpDashboardPresenter class >> defaultSpec [
 	^ SpBoxLayout newVertical
 		add: #currentSessionInfoPresenter;
 		add: (SpBoxLayout newHorizontal
 				add: #enableMonitorButtonPresenter;
 				add: #createLogButtonPresenter;
 				add: #openSettingsButtonPresenter;
 				yourself)
			expand: false
			fill: false
			padding: 0;
		yourself
```
Please, can you fix this?

Comment (jecisc) [reply src/EpiceaBrowsers/EpDashboardPresenter.class.st:24]: Done in https://github.com/pharo-project/pharo/pull/4474

**Pull request merged**